x/merkledb/metrics.go

```diff
@@ -137,15 +157,25 @@ func newMetrics(namespace string, reg prometheus.Registerer) (merkleMetrics, err
 			Name:      "hashes_calculated",
 			Help:      "cumulative number of node hashes done",
 		}),
-		dbNodeCacheHit: prometheus.NewCounter(prometheus.CounterOpts{
+		dbValueNodeCacheHit: prometheus.NewCounter(prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "db_value_node_cache_hit",
```

Comment:
Same as above here about leading `db_` -- I think `value_node_cache_hit` would suffice, for example